hms-app/src/components/HackathonHeader.tsx

```diff
@@ -21,25 +21,29 @@ export default function HackathonHeader(props: IProps) {
 
   return (
     <Container my={30}>
-      <Center>
-        <Title>{hackathonData.title}</Title>
-      </Center>
-      <Center mt={10}>
-        <Badge
-          size={'lg'}
-          variant={'gradient'}
-          gradient={
-            theme.colorScheme === 'dark'
-              ? { from: 'teal', to: 'blue', deg: 60 }
-              : { from: '#ed6ea0', to: '#ec8c69', deg: 35 }
-          }
-          radius={'xs'}
-        >{`${new Date(
-          hackathonData.startDate
-        ).toLocaleDateString()} - ${new Date(
-          hackathonData.endDate
-        ).toLocaleDateString()}`}</Badge>
-      </Center>
+      {hackathonData.title && (
```

Comment:
in the backend we ensure that a title is never empty so you could remove this check 🤔

Reply:
that, of course, means you need to ensure that any other source of the data also is correct